int32 robot_id
---
bool is_ready
float32[] point
float32[] normal